core/dbt/events/types.py

```diff
@@ -1785,22 +1791,32 @@ def message(self) -> str:
 
 @dataclass
 class PrintStartLine(InfoLevel, Cli, File):
+    # TODO: add node info
+    # report_node_data: ParsedModelNode
     description: str
     index: int
     total: int
+    status: str = 'RUN'
```

Comment:
this doesn't inherit from `NodeInfo`. What's this status for?

Reply:
This is literally for the output in `format_fancy_output_line`.  I pulled it out originally to share values but shouldn't as it will change the text.  Changing messages text is beyond the scope of this specific task so I'll change it back to how it was to reduce confusion.

Reply:
ahh. makes sense. might be worth it to drop a TODO to that effect.